docker/Dockerfile

```diff
@@ -1,19 +1,81 @@
-# To build the Docker container, run the following lines from your proteus root directory:
-# cd docker
-# docker build --rm -t proteus:latest .
+# Use Ubuntu as the base image
+FROM ubuntu:22.04
 
-# Get latest base MOOSE image
-FROM idaholab/moose:latest
+# Set environment variables
+ENV DEBIAN_FRONTEND=noninteractive
+ENV PROTEUS_DIR=/proteus
```

Comment:
The proteus installation is in `/proteus` but the working directory we enter at the end is `/opt/proteus` (see ln 77). This is confusing. The user experience is currently to enter a directory called proteus, run `ls` and be faced with an empty directory. Then the user may find by moving through the directory structure that the actual installation is in another directory called proteus in a different location. Is this a typo and both are meant to be `/opt/proteus`?